tests/test_feature_aggregate.py

```diff
@@ -53,6 +52,20 @@
     columns=["variable", "unit"] + TEST_YEARS,
 )
 
+NEG_WEIGHTS_DF = pd.DataFrame(
```

Comment:
I created an additional `NEG_WEIGHTS_DF` as input data, because the initial usage of the simple_df and appending it with a data set with a negative value led to a more complicated testing function.

Reply:
Nice!